messagecard.go

```diff
@@ -160,14 +158,13 @@ func (mc *MessageCard) AddSection(section ...*MessageCardSection) error {
 		// field in the output JSON.
 		// See also https://github.com/golang/go/issues/11939
 		switch {
-
 		// If any of these cases trigger, skip over the `default` case
 		// statement and add the section.
 		case s.Images != nil:
 		case s.Facts != nil:
 		case s.HeroImage != nil:
-		case s.StartGroup != false:
-		case s.Markdown != false:
+		case s.StartGroup:
+		case s.Markdown:
```

Comment:
@dasrick I completely get why this was changed. I started adding checks at the top of the switch statement and just went down from there only changing up the expected values as I went. Good catch.